app/assets/javascripts/analytics/_init.js

```diff
@@ -5,6 +5,7 @@
     this.register();
     this.pageViews.init();
     this.events.init();
```

Comment:
So does this work / is it initialising the FE toolkit events.js, and if so shouldn't that be happening in `toolkit/javascripts/analytics/_init.js`?

Reply:
Hi @galund - yes this is needed as this is an override function. If the init is not done here then the function would not trigger.

Reply:
Comment here - and perhaps also in the effectively-overridden init function? (in the other repo I guess) - and then we're good. I'll approve now so you don't have to wait for me any more.

Reply:
Comment left here

https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/443#issuecomment-398343433